src/nccl_ofi_scheduler.c

```diff
@@ -22,6 +22,40 @@ static inline size_t sizeof_schedule(int num_rails)
 		+ num_rails * sizeof(nccl_net_ofi_xfer_info_t);
 }
 
+/*
+ * @brief  This function calculates the optimal number of stripes
+ * for the payload size based on the min_stripe_size.
+ * It first determines the initial number of stripes based on the message size
+ * and a minimum stripe size, ensuring that the result is at least 1 and at
+ * most the number of available rails. The function then adjusts the number of
```

Comment:
This does not match what the commit message says.  The code does not "modify the num_stripes to be rounded off to the nearest factor of the num_rails."

Reply:
done. I have updated the commit message and the function description.